### DISCUSSION TITLE
Eufy Security: Setup "Unknown error occurred"

### DISCUSSION THREAD

**Discussion by kaicolo679** [fuatakgun/eufy_security#864] [Q&A]
<img width="472" alt="Screenshot 2023-07-31 at 10 48 11 PM" src="https://github.com/fuatakgun/eufy_security/assets/140864565/bbdde236-2ae1-4bbc-9bc4-92295dac0b12">

Hi guys, 
Hope someone can help me with this. 
I've searched everywhere but unable to move forward from here. 

TIA!

**Comment by NeriedAU** [comment]
Can you confirm you have [eufy-security-ws installed](https://github.com/bropat/hassio-eufy-security-ws ), configured and running? 

Once you have added the Add On repo and installed eufy-secrutiy-ws through the Add On Store, you will then need to configure the add on with your login details. Check the logs for the Add On once you have configured it to make sure it is all good and your homebase has been discovered then try the Eufy Security integration add again.
